src/components/m-header-languages/_template.js

```diff
@@ -22,10 +22,11 @@ export default ({ value, items }) => [html`
         <a
           data-index="${name}"
           data-selected="${name === value ? 'true' : 'false'}"
-          class="${classnames('m-header-languages__list-link', {
+          class="${classnames('m-header-languages__list-link', 'js-header-languages__list-link', {
             'is-header-languages-active': isActive,
           })}"
           href="${url}"
+          lang="${name}"
```

Comment:
@LucaMele 
Hmh, may this should be refactored to have also `code` in addition to `name`